indico/web/client/styles/modules/_dashboard.scss

```diff
@@ -82,20 +82,23 @@
   background: #dcebf5;
 }
 
-.dashboard-box ol {
+.dashboard-box ol,
+.dashboard-box ul {
```

Comment:
I'd change the other `ol` in the dashboard template as well instead of defining the rule for both `ol` and `ul`..

Reply:
While at it, I increased the width of the date field so it does not wrap on `last Wednesday` entry:

![a](https://user-images.githubusercontent.com/6058151/133216829-2d2415e3-271c-4c9e-8968-b240ba6ce5a6.png)